packages/@okta/vuepress-site/docs/guides/create-token-with-groups-claim/overview/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Overview
+---
+Tokens contain claims that are statements about the subject, such as name, role, or email address. Beyond the [default set of claims](https://developer.okta.com/docs/guides/validate-id-tokens/) that are contained in ID tokens and access tokens, you can define your own custom claims and group claims. For example, you might want to add a user's email address to an access token and use that to uniquely identify the user, or you may want to add information stored in a user profile to an ID token. You can also add a groups claim to ID tokens for any combination of application groups and user groups to perform SSO using the Okta Org Authorization Server. And, you can add a groups claim to ID tokens and access tokens to perform authentication and authorization using a Custom Authorization Server.
```

Comment:
Link to the Authorization server concept piece here where we specify `Okta Org Authorization Server` and `Custom Authorization Server`